container-images/intel-gpu/Containerfile

```diff
@@ -1,10 +1,10 @@
-FROM quay.io/fedora/fedora:41 as builder
```

Comment:
Did you test this build? Fedora 42 upgrade broke some of our builds last time, may not have been intel-gpu though

Reply:
Containers were building, not tested other thing.

Reply:
$ REGISTRY_PATH=quay.io/ramalama_rpc time ./container_build.sh build

Successfully tagged quay.io/ramalama_rpc/rocm-ubi-rag:latest
2320d96ab988d76861833be1cab49de9a886d2f02867885ff2d297669fb38f2e
51689.13user 2185.82system 2:19:03elapsed 645%CPU (0avgtext+0avgdata 2345616maxresident)k
25355192inputs+887504640outputs (6412major+468807028minor)pagefaults 0swaps


Current PR makes **all** container build.